test/common_test/free_joint_features.cc

```diff
@@ -240,6 +240,61 @@ TEST_F(FreeGroupFeaturesTest, NestedFreeGroupSetWorldPose)
   }
 }
 
+TEST_F(FreeGroupFeaturesTest, FreeGroupNestedNoLink)
+{
+  const std::string modelStr = R"(
+    <sdf version="1.11">
+      <model name="box">
+        <pose>1 2 3.0 0 0 0</pose>
+        <model name="nested_box"/>
```

Comment:
This is not valid SDF because a non-static model has to have at least one link. Recommend adding `<static>true</static>` to the nested model.